mergify_engine/delayed_refresh.py

```diff
@@ -110,5 +116,7 @@ async def send(
                 "source": "delayed-refresh",
             },  # type: ignore[typeddict-item]
         )
+        keys_to_delete.add(subkey)
 
-        await redis_cache.zrem(DELAYED_REFRESH_KEY, subkey)
+    await pipe.execute()
+    await redis_cache.zrem(DELAYED_REFRESH_KEY, *keys_to_delete)
```

Comment:
why not putting this in the pipe also?

Reply:
that's not the same DB and it's just one redis call